README.md

```diff
@@ -1,28 +1,109 @@
+Arby is a market maker bot that allows its users to easily arbitrage between [OpenDEX](http://opendex.network/) and centralized exchanges. Users with spare capital are able to "earn interest" on their assets by providing liquidity to the OpenDEX network.
 
-Arby looks for arbitrage opportunities on the OpenDEX network. It will enter active orders with the configured margin to the orderbook and updates them as soon as the price changes.
+The purpose of Arby is to pull liquidity from centralized exchanges into OpenDEX while incentivizing liquidity providers (market makers) financially.
+
+### How it works
+Arby looks for arbitrage opportunities on the OpenDEX network. It will issue orders with the configured margin to the orderbook and update them as soon as the price changes.
+
+Trades on OpenDEX are settled in seconds. This creates a potential arbitrage opportunity between centralized exchanges and OpenDEX.
+
+Example with real numbers:
+
+- Balances before starting Arby
+
+| Currency | OpenDEX | Binance |
+| -------- | ------- | ------- |
+| BTC      | 1.0     | 1.0     |
+| USDT     | 11000   | 11000   |
+
+- Total BTC balance between OpenDEX and Binance `1.0 + 1.0 = 2.0 BTC`
+- Total USDT balance between OpenDEX and Binance `11000 + 11000 = 22000 USDT`
+
+---
+- Arby is started for BTC/USDT trading pair with 3% margin. The configured centralized exchange is Binance.
+- Balance on OpenDEX is 1 BTC and 1 BTC on Binance
+- USDT balance on OpenDEX is 10000 USDT and 10000 USDT on Binance
+- Arby monitors the latest price of BTC/USDT on Binance. The latest price is $10000.
+- Arby creates/updates the buy order on OpenDEX for `10000 - 0.03 * 10000 = $9700`.
+- Arby creates/updates the sell order on OpenDEX for `10000 * 1.03 = $10300`.
+- Whenever the latest price on Binance changes, Arby updates the orders on OpenDEX accordingly.
+
+##### Scenario A
+- OpenDEX buy order is filled for 1 BTC ($9700).
+- Arby issues a sell order on Binance for 0.97 BTC ($9700), locking in 0.03 BTC ($300) profit.
+- Balances after scenario A
+
+| Currency | OpenDEX | Binance |
+| -------- | ------- | ------- |
+| BTC      | 2.0     | 0.03    |
+| USDT     | 1300    | 20700   |
+
+- Total BTC balance between OpenDEX and Binance `2.0 + 0.03 = 2.03 BTC` (0.03 BTC yield)
+- Total USDT balance between OpenDEX and Binance `1300 + 20700 = 22000 USDT`
+
+---
+
+##### Scenario B
+- OpenDEX sell order is filled for 1 BTC ($10300)
+- Arby issues a buy order on Binance for 1.03 ($10300), locking in 0.03 BTC ($300) profit.
+
+| Currency | OpenDEX | Binance |
+| -------- | ------- | ------- |
+| BTC      | 0       | 2.03    |
+| USDT     | 21300   | 700     |
+
+Total BTC balance between OpenDEX and Binance `2.0 + 0.03 = 2.03 BTC` (0.03 BTC yield)
+Total USDT balance between OpenDEX and Binance `21300 + 700 = 22000 USDT`
+
+---
+
+### FAQ
+
+#### Can I use Kraken instead of Binance?
```

Comment:
Which exchanges are supported?
Kraken and Bitfinex are next on the roadmap, link to issues.